templates/index.html

```diff
@@ -0,0 +1,7 @@
+<form method="POST" ACTION="">
```

Comment:
I assume this is for registering? Or is it for logging in? If for logging in, leave it as it is for now.